Formula/mariadb-connector-odbc.rb

```diff
@@ -0,0 +1,25 @@
+class MariadbConnectorOdbc < Formula
+  desc "MariaDB Standardized database driver"
+  homepage "https://downloads.mariadb.org/connector-odbc/"
+  url "http://nyc2.mirrors.digitalocean.com/mariadb//connector-odbc-3.0.2/mariadb-connector-odbc-3.0.2-ga-src.tar.gz"
```

Comment:
can we get an https link?

Reply:
Sadly it doesn't seem like it -- the server's unresponsive when I set it to https://.  A workaround is to use MariaDB's site as a proxy (quote, unquote), though it may not look the best, nor function as we expect:

https://downloads.mariadb.org/f/connector-odbc-3.0.2/mariadb-connector-odbc-3.0.2-ga-src.tar.gz/from/http%3A//mirrors.accretive-networks.net/mariadb/?serve

Reply:
It works for me. I'd try `https://downloads.mariadb.org/f/connector-odbc-3.0.2/mariadb-connector-odbc-3.0.2-ga-src.tar.gz` through CI